8_visualize_metrics.yml

```diff
@@ -0,0 +1,220 @@
+target_default: 8_visualize
```

Comment:
Change this to match the file name `8_visualize_metrics` (and below anytime it's mentioned)